src/fastify-plugins/utils.js

```diff
@@ -43,16 +43,3 @@ export async function getFastifyServerAddress(server, { timeout } = {}) {
 
   return 'http://' + addr
 }
-
-/**
- * @param {Readonly<Date>} lastModified
- */
-export function createStyleJsonResponseHeaders(lastModified) {
-  return {
-    'Cache-Control': 'max-age=' + 5 * 60, // 5 minutes
-    'Access-Control-Allow-Headers':
-      'Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since',
-    'Access-Control-Allow-Origin': '*',
-    'Last-Modified': lastModified.toUTCString(),
-  }
-}
```

Comment:
this function is no longer used but does imply that we used to do the right thing 😄 wondering if any of these headers should be added as well 🤔

Reply:
not sure, as long as it works without them 🤷 
I think we should (outside this PR) figure out caching headers and last-modified so that if we change the style then it overrides the cache, but I think using without a cache is fine for now.